src/main/java/com/aspire/student/util/CourseCategoryUtil.java

```diff
@@ -25,6 +25,7 @@
   public static void loadCourseCategoryList(CourseCategoryService courseCategoryService,
       Map<String, Object> map) {
     List<CourseCategory> courseCategoryList = null;
+     List<CourseCategory> courseCategoryList200 = null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "courseCategoryList200" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)